client/src/main/java/pro/panopticon/client/model/Measurement.java

```diff
@@ -7,16 +7,22 @@
     public String status;
     public String displayValue;
     public CloudwatchValue cloudwatchValue;
+    public String description;
 
-    public Measurement(String key, String status, String displayValue, CloudwatchValue cloudwatchValue) {
+    public Measurement(String key, String status, String displayValue, CloudwatchValue cloudwatchValue, String description) {
```

Comment:
Er ikke alltid vi bruker `CloudwatchValue`, så tar gjerne en konstruktør uten den men med `description` 😊

Skulle vi deprecated de som ikke har `description` for å nudge oss til å ta det i bruk, eller blir det for drastisk i første omgang? 🙈